robo-script-7.1-and-above.json

```diff
@@ -2376,15 +2367,6 @@
         "resourceId": "hashengineering.darkcoin.wallet:id/home_button_view",
         "contentDescription": ""
 
-      },
-      {
-        "className": "de.schildbach.wallet.ui.widget.GlobalFooterView",
-        "recyclerViewChildPosition": -1,
-        "adapterViewChildPosition": -1,
-        "groupViewChildPosition": 0,
-        "resourceId": "",
-        "contentDescription": ""
-
```

Comment:
Is this file auto-generated?

Reply:
This file was created by Android Studio as a Robo script after recording steps to run in Firebase TestLab.  It worked well for version v7.1 and a few releases after that.  

It is no longer usable and we can delete it.

Reply:
This file can be deleted.